src/database/tables.py

```diff
@@ -5,6 +5,7 @@
 
 Base = declarative_base()
 
+# This may be deprecated
```

Comment:
How so?

Reply:
it's only necessary for timestamps of when we get the data, not necessary for Reddit imo